src/azure-openapi-linter/rules/ArraySchemaMustHaveItems.ts

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { MergeStates, OpenApiTypes, rules } from '../rule';
+export const ArraySchemaMustHaveItems: string = "ArraySchemaMustHaveItems";
+
+const jp = require('jsonpath');
+
+rules.push({
+  id: "S2009",
+  name: ArraySchemaMustHaveItems,
+  severity: "warning",
```

Comment:
is this the violation that would result in AutoRest producing a code generation error?